### DISCUSSION TITLE
holding migrations for source project in a fork

### DISCUSSION THREAD

**Discussion by khodedawsh** [sqlalchemy/alembic#1477] [Usage Questions] [answered]
Hi, I have forked a project, changed the database model significantly and created a new migration sequence.
Now I'd like to have migrations from that software, that I could run on databases from the upstream.

it would be great in case `alembic upgrade head`  works with a blank schema but also when the revision is from the upstream.

**Comment by zzzeek** [comment, marked as answer]
you should define your new migrations in terms of those upstream migrations, then, which would remain present in your project

**Reply by khodedawsh** [reply]
No way to separate them then?

**Reply by zzzeek** [reply]
you can have the migrations in multiple directories if you want, see version_locations at https://alembic.sqlalchemy.org/en/latest/tutorial.html#editing-the-ini-file
